Demos/DemoLightbox/Podfile

```diff
@@ -3,6 +3,9 @@ platform :ios, '8.0'
 use_frameworks!
 inhibit_all_warnings!
 
-pod 'Lightbox', path: '../../'
-pod 'Sugar'
-pod 'Hue', git: 'https://github.com/hyperoslo/Hue'
+target 'DemoLightbox' do 
+	pod 'Lightbox', path: '../../'
+	pod 'Sugar'
+	pod 'Hue', git: 'https://github.com/hyperoslo/Hue'
```

Comment:
I think we should remove these dependencies. As far as i know `Sugar` and `Hue` are being used for two functions, `delay` and `UIColor.hex`. 

Although i prefer to reuse code but at the same time i think libs should be modular and try to achieve 0 dependencies (not possible always but that should be the goal)